### PR TITLE
feat(frontend): UI component for content with buttons

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

We create an UI component that organize the content of the modals that have a button at the end, to concentrate the code.

### CODE REVIEW

File: src/frontend/src/lib/components/ui/ContentWithButtons.svelte [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<slot class="stretch" />
+
+<slot name="buttons" />
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/ui/ContentWithButtons.svelte:3]: What do you think about naming this slot `toolbar` or something similar? If yes, don't know if the cmp should reflects it?

Just an idea, already approving the PR.

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: src/frontend/src/lib/components/ui/ContentWithButtons.svelte (hunk 1 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/ui/ContentWithButtons.svelte:3]: So, if it's `toolbar` the component will be `ContentWithToolBar`, correct?

Comment (peterpeterparker) [reply src/frontend/src/lib/components/ui/ContentWithButtons.svelte:3]: Ah why not, sounds cool no?

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/ui/ContentWithButtons.svelte]: Yes, better! Eheh! Changed

**Pull request merged**